pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -862,6 +862,16 @@ type CommonPrometheusFields struct {
 	// +optional
 	TSDB *TSDBSpec `json:"tsdb,omitempty"`
 
+	// The name of the service name used by the underlying StatefulSet(s) as the governing service.
+	// If defined, it must be created before the Prometheus/PrometheusAgent resource in the same namespace and it must define a selector that will match the pod labels.
```

Comment:
(nit) just to be explicit
```suggestion
	// If defined, the Service must be created before the Prometheus/PrometheusAgent resource in the same namespace and define a selector that matches the pod labels.
```